addons/account/models/account_payment_term.py

```diff
@@ -258,7 +258,16 @@ class AccountPaymentTermLine(models.Model):
             ('days_after', 'Days after invoice date'),
             ('days_after_end_of_month', 'Days after end of month'),
             ('days_after_end_of_next_month', 'Days after end of next month'),
+            ('days_end_of_month_on_the', 'Days end of month on the'),
         ], required=True, default='days_after')
+    display_days_next_month = fields.Boolean(compute='_compute_display_days_next_month')
+    days_next_month = fields.Char(
+        string='Days on the next month',
+        readonly=False,
+        store=True,
```

Comment:
```suggestion
```

Isn't this the default for non-computed fields?

Reply:
I will change that :eyes: